constant.json

```diff
@@ -0,0 +1,55 @@
+{
+    "id": "constant",
+    "summary": "Define a constant value",
+    "description": "Define a constant value that can be reused in multiple places of the process graph",
+    "categories": [
+        "math > constants"
+    ],
+    "parameters": [
+        {
+            "name": "x",
+            "description": "The value of the constant.",
+            "schema": {
+                "type": "number"
+            }
+        }
+    ],
+    "returns": {
+        "description": "The value of the constant.",
+        "schema": {
+            "type": "number"
+        }
+    },
+    "examples": [
```

Comment:
Support for process graph examples has been removed. Such examples use links now.

Reply:
I also should find a better example than the one with "margin" because parameters feel more suited for it than a constant indeed